extraction/content_extractors/image_extractor.py

```diff
@@ -26,7 +26,11 @@ def __create_image(self, node, caption_tag=None):
         """This method extracts all attributes for the image
         and creates Image object instance"""
 
-        image_url = node['src']
+        if node.has_attr('data-src'):
+            image_url = node['data-src']
+        else:
+            image_url = node['src']
+
```

Comment:
Can you add comment explaining this logic?

Reply:
Makes sense! Updated!